curvefs/proto/topology.proto

```diff
@@ -90,9 +89,13 @@ message MetaServerData {
     required string externalIP = 6;
     required uint32 externalPort = 7;
     required uint32 serverId = 8;
-    required uint64 diskCapacity = 9;
-    required uint64 diskUsed = 10;
-    required uint64 memoryUsed = 11;
+    required heartbeat.MetaSeverSpaceState spaceState = 9;
+    // required uint64 diskThresholdByte = 9;
+    // required uint64 diskCopysetMinRequireByte = 10;
+    // required uint64 diskUsedByte = 11;
+    // required uint64 memoryThresholdByte = 12;
+    // required uint64 memoryCopySetMinRequireByte = 13;
+    // required uint64 memoryUsedByte = 14;
```

Comment:
Do you keep these fields for any purpose?

Reply:
done